custom/ewsghana/alerts/alerts.py

```diff
@@ -90,17 +94,17 @@ def stock_alerts(transactions, verified_number):
     if super_message:
         stripped_message = super_message.strip().strip(';')
         super_message = _('Dear %s, %s is experiencing the following problems: ') + stripped_message
-        send_message_to_admins(user, super_message.rstrip())
+        send_message_to_admins(user, sql_location, super_message.rstrip())
     send_sms_to_verified_number(verified_number, message.rstrip())
 
 
-def send_message_to_admins(user, message):
-    users = get_users_by_location_id(user.domain, user.location.get_id)
-    in_charge_users = [
-        u
-        for u in users
-        if get_verified_number_for_recipient(u) and "In Charge" in u.user_data.get('role', [])
-    ]
+def send_message_to_admins(user, sql_location, message):
+    in_charge_users = map(CommCareUser.wrap, iter_docs(
+        CommCareUser.get_db(),
+        [in_charge.user_id for in_charge in user.sql_location.facilityincharge_set.all()]
+    ))
     for in_charge_user in in_charge_users:
-        send_sms_to_verified_number(get_verified_number_for_recipient(in_charge_user),
-                                    message % (in_charge_user.full_name, in_charge_user.location.name))
+        verified_number = get_verified_number_for_recipient(in_charge_user)
```

Comment:
Is it intentional to only be sending to verified numbers here? We could also use `get_preferred_phone_number_for_recipient` which would allow us to send to unverified (or duplicated) numbers as well, but I didn't know what the requirement was here.